compose/service/settings.go

```diff
@@ -0,0 +1,97 @@
+package service
+
+import (
+	"context"
+
+	_ "github.com/joho/godotenv/autoload"
```

Comment:
You do not need this here anymore. Was used for auth settings autodiscovery on system (and this import shoud/cloud be moved there too)